### PR TITLE
docs: Updated docs to retrieve auth token

### LINKED ISSUE #289: Feature: Copy auth token in navbar
### Type of feature

🍕 Feature

### Current behavior

Currently, we have a "Print auth token button" which prints the token to devtools console. But this has been updated to "Copy auth token button" in https://github.com/open-sauced/hot/pull/514.

### Suggested solution

Update the "[Set Up Authentication](https://docs.opensauced.pizza/contributing/set-up-authentication/)" page on docs to reflect this new changes.

### Additional context

https://github.com/open-sauced/hot/pull/514

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OmkarPh**
## Description
This PR updates the docs from "Print auth token" to "Copy auth token" based on changes at https://github.com/open-sauced/hot/pull/514.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #289 
Implemented in https://github.com/open-sauced/hot/pull/514

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by adiati98** [comment]
Hi @OmkarPh 

First of all, thank you for the PR! ✨️
I see that this PR is related to an issue in our hot repo.

However, an issue to update the docs is required for you to make any changes in this repo. Can you please create an issue for the purpose of updating the docs?

Based on [our guidelines](https://docs.opensauced.pizza/contributing/introduction-to-contributing/#pull-requests-pr), PRs without link to any issue will be closed. And for that reason, we're closing this PR.

Feel free to create a new PR or ask us to reopen this PR once there's an issue accompanying the PR.

Also, we would like to confirm this changes/update from the @open-sauced/engineering team.

Thank you! 😊

**Comment by adiati98** [comment]
Hi @OmkarPh,

I've reopened your PR.

Please update the PR form as follow:

- Let's make the description more descriptive:

   > This PR updates the docs from "Print auth token" to "Copy auth token" based on changes at https://github.com/open-sauced/hot/pull/514.
- Link your issue in the "Related Tickets and Documents" section instead of description.
- As you can see, there's a bot here mentioning [issue reference](https://github.com/open-sauced/docs/pull/280#pullrequestreview-1943601660) as part of PR complience. That's because GitHub only receives [particular keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). So, please use the prefix `Closes` when linking your issue.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @OmkarPh,

I left a review here to be fixed. Thanks! 🙂

File: docs/contributing/set-up-authentication.md [hunk 1]
```diff
@@ -22,15 +22,11 @@ The following steps outline how to obtain an authentication token from the [hot.
 
    ![Avatar in top right corner of the page](../../static/img/contributing-avatar-top-right.png)
 
-2. Click on the **Print auth token** option. Don't worry, this is a safe operation. The token is only printed to the console.
+2. Click on the **Copy auth token** option to copy your auth token to clipboard
```
[the remark this answers is not in the document]
Comment (OmkarPh) [reply docs/contributing/set-up-authentication.md]: Thanks for the review !

I've made the changes

**Review by adiati98 — APPROVED** [review]
It looks good to me.
But I would like to have @BekahHW and @nickytonline second opinion on the content. 🙂

**Review by BekahHW — APPROVED** [review]

**Pull request merged**